### PR TITLE
add feature request template

### PR DESCRIPTION
**Pull request by jellizaveta**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: Feature request
+description: Suggest an idea for AdGuard Filters
```
Comment (Alex-302) [inline .github/ISSUE_TEMPLATE/feature_request.yml:2]: Wrong repo

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+labels: ["Feature Request"]
```
Comment (Alex-302) [inline .github/ISSUE_TEMPLATE/feature_request.yml:3]: What about to add `Infrastructure` label?

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 2 above)
Comment (jellizaveta) [reply .github/ISSUE_TEMPLATE/feature_request.yml:3]: Do you mean change the name from feature request to Infrastructure ?

Comment (Alex-302) [reply .github/ISSUE_TEMPLATE/feature_request.yml:3]: I thought maybe two labels would be better for this template. But for now, it seems like one is enough.
`labels: ["Feature Request", "Infrastructure"]`

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**